include/sys/mem_manage.h

```diff
@@ -9,6 +9,9 @@
 
 #include <sys/util.h>
 #include <toolchain.h>
+#if defined(CONFIG_ARM_MMU) && defined(CONFIG_ARM64)
```

Comment:
My unclear comment may mislead you, but I mean something like:
`#if defined(CONFIG_ARM64) && (defined(CONFIG_ARM_MMU) || defined(CONFIG_ARM_MPU) )`
or just a single condition:
`#if defined(CONFIG_ARM64) ` since it's the same for both MMU and MPU system.
Any ideas?

Reply:
Per my understand, firstly we need to check CONFIG_ARM_MMU because only used in arch/arm64/core/mmu.c, secondly this is ARM64 features, we these two condition should both be matched, right?

Reply:
Right, I got what you mean. LGTM.
I had thought it could be extended to MPU but suddenly realized only in MMU for now. Thanks, I'll take another look into it MPU case if possible